level3/[level3] 이도형/java-racing-car/src/main/java/Controller/GameController.java

```diff
@@ -0,0 +1,22 @@
+package Controller;
+
+import Model.GameModel;
+import View.GameConsoleView;
+
+public class GameController implements CarApplication {
```

Comment:
인터페이스와 해당 인터페이스를 구현한 클래스명의 추상화 정도를 잘 나타내야 합니다.
인터페이스를 구현한 클래스는 해당 인터페이스보다 조금 구체적으로 나타내야 좋습니다
```suggestion
public class CarGameController implements CarApplication {
```